python/manual-instrumentation/server.py

```diff
@@ -0,0 +1,70 @@
+from time import sleep
+
+from flask import Flask
+from opentelemetry import trace
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import (
+    OTLPSpanExporter,
+)
```

Comment:
Is there a reason to use the parenthesized form here, instead of formatting like the rest of the imports?

Reply:
Ohh sorry, my mistake. Initially, I had multiple things to import from the library. Forgot to remove the parenthesis. I will work on it.